modules/src/movie.py

```diff
@@ -8,7 +8,7 @@ def process(input, entities):
         r = requests.get('http://www.omdbapi.com/?t=' + movie + '&plot=short&r=json')
         data = r.json()
         output['input'] = input
-        output['output'] = data['Title'] + '\nPlot: ' + data['Plot'] + '\nIMDb Rating: ' + data['imdbRating']
+        output['output'] = data['Title'] + '\nPlot: ' + data['Plot'] + '\nIMDb Rating: ' + data['imdbRating'] + data['Year']
```

Comment:
We want to make this PR about weather only (so that someone can refer it in the future for adding another module). Can you make another PR with this change?